pkg/kapp/cmd/app/delete.go

```diff
@@ -211,7 +235,7 @@ func (o *DeleteOptions) calculateAndPresentChanges(existingResources []ctlres.Re
 		changeSetView.Print(o.ui)
 	}
 
-	return clusterChangeSet, clusterChangesGraph, (len(clusterChanges) == 0), nil
+	return clusterChangeSet, clusterChangesGraph, changeFlags{len(clusterChanges) == 0, isFullyDeleteApp}, nil
```

Comment:
nit: when building a struct with fields of the same type, it's better to use field names to make sure we are setting correct fields.

Reply:
Done